eleventy.config.js

```diff
@@ -125,6 +127,63 @@ module.exports = function(config) {
 
     // Трансформации
 
+    config.addTransform('optimizeContentImages', async function(content) {
+        if (!this.page.inputPath.includes('/articles/')) {
+            return content;
+        }
+
+        if (!this.page.outputPath.endsWith('.html')) {
+            return content;
+        }
+
+        const { document } = parseHTML(content);
+        const images = Array.from(document.querySelectorAll('.article__content img'))
+            .filter((image) => !image.src.startsWith('http://') && !image.src.startsWith('https://'));
```

Comment:
```suggestion
            .filter((image) => !image.src.match(/^https?/));
```

Не настаиваю, но кажется чуть легче читается, что скажешь? Хотя с этими регулярками кому как :)

Reply:
Ок

Reply:
Можно подушню? (а я всё равно подушню 😈) 

Протокол `http` всё равно неактуален, браузер не будет загружать такие картинки. Так что можно просто оставить один `startsWith`. 

Регулярки это сложно и этот фикс а очередной раз доказывает это утверждение. 

Мне жалко картинку с именем `https-is-awesome.png` 😅

Reply:
Спасибо, поправил